### DISCUSSION TITLE
Change default font size for notes

### DISCUSSION THREAD

**Discussion by knowine** [freeplane/freeplane#1338] [Help] [answered]
How can I modify and set a new default font size for notes? 
![image](https://github.com/freeplane/freeplane/assets/90181395/84a8f30b-4f90-4690-bd5d-332e9bfd6783)

**Comment by sifran-github** [comment]
in styles editor (ctrl+f11), change the style of the "note" node.

**Reply by knowine** [reply]
Thank you, it worked! But I noticed the change isn't applied to other maps as well. How do I obtain that?

**Reply by sifran-github** [reply, marked as answer]
As far as I know, There is no way to define this globally. styles settings are map specific settings.

But, you can create a template and use it for all your maps.
You can put the template in the directory in tools->open user directory, and then "templates". create this directory if it doesn't exist.
Then you can choose this as your default template for all your maps.

also, after you created it, you can apply it to existing maps by going to: format->styles->copy map styles from...

you can also choose the default template in :
Tools->Preferences->Environment->Files->Standard template file
and there is also an option to skip the template selection window.
So this template is effectively on every new map that you create.

more info:
https://docs.freeplane.org/user-documentation/templates.html

**Reply by knowine** [reply]
Thank you! It worked!
